src/test/groovy/com/jvm_bloggers/entities/blog_post/BlogPostSpec.groovy

```diff
@@ -10,14 +8,17 @@ import spock.lang.Unroll
 import java.time.LocalDateTime
 import java.time.Month
 
+import static com.jvm_bloggers.ObjectMother.aBlogPost
+import static java.time.LocalDateTime.of
```

Comment:
Static import makes sense when by reading only a method name we can easily tell what method is being called. When I see `of(...)` in the code, it's hard to immediately know "Ahh, it is from `LocalDateTime`" :) So I would get rid of this import.